contents/docs/session-replay/_snippets/android-installation.mdx

```diff
@@ -0,0 +1,50 @@
+> 🚧 **NOTE:** Android is considered `beta` and is free while in beta. We are keen to gather as much feedback as possible so if you try this out please let us know. You can send feedback via the [in-app support panel](https://us.posthog.com#panel=support%3Afeedback%3Asession_replay%3Alow) or one of our other [support options](/docs/support-options).
+
+## Installation 
```

Comment:
We don't need this header since the page header already says "Session replay installation". Repeat for other platforms too
```suggestion
```

Reply:
Actually, lets do this as a "Step one" and "step two"
```suggestion
## Step one: Add PostHog to your app 
```